include/functions.h

```diff
@@ -1529,22 +1529,23 @@ void* GameAlloc_Malloc(GameAlloc* this, u32 size);
 void GameAlloc_Free(GameAlloc* this, void* data);
 void GameAlloc_Cleanup(GameAlloc* this);
 void GameAlloc_Init(GameAlloc* this);
-// ? func_800C5850(?);
-// ? func_800C58BC(?);
-// ? func_800C5AB0(?);
-// ? func_800C5B14(?);
-// ? func_800C5C2C(?);
-// ? func_800C5D08(?);
-// ? func_800C5D94(?);
-// ? func_800C5DC0(?);
-// ? func_800C61D8(?);
+void Graph_FaultClient();
+void Graph_DisassembleUCode(void* arg0);
+void Graph_UCodeFaultClient(void* arg0);
+void* Graph_InitTHGA(GraphicsContext* gfxCtx);
+GameStateOverlay* Graph_GetNextGameState();
+void Graph_Ct(GraphicsContext* gfxCtx);
+void Graph_Dt(GraphicsContext* gfxCtx);
```

Comment:
I think init and destroy would be more consistent with the rest of the codebase.